spec/features/user_sorts_users_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+feature "User searches for another user", %q{
+  As a student and teacher
+  I want to find other users by name
+  So that I can view there profiles
+
+  Acceptance Criteria:
+  - [ ] The user index page will first include all users
+  - [ ] I can type in a users name and it will update the
+  user index page with
+  } do
+
+    #add autocomplete for skills column as time allows
+
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      sign_in(@user)
+    end
+
+    scenario "User sucessfullly finds who they are looking for" do
+      #create the skills Ruby and Javascript
+      skill1 = Skill.create(name: "Ruby")
+      skill2 = Skill.create(name: "Javascript")
+
+      #create a bunch of users to be sorted
+      user1 = FactoryGirl.create(:user)
+      user2 = FactoryGirl.create(:user)
+      user3 = FactoryGirl.create(:user)
+      user4 = User.create(username: "nicole", email: "someemail@gmail.com", password:"password89")
+
+      users_had_skill = HadSkill.create(skill_id: skill1.id, user_id: @user.id)
+      users_wanted_skill = WantedSkill.create(skill_id: skill2.id, user_id: @user.id)
```

Comment:
Useless assignment to variable - `users_wanted_skill`.<br>Line is too long. [85/80]